src/components/v5/common/CompletedAction/partials/PaymentBuilder/partials/PaymentBuilderWidget/PaymentBuilderWidget.tsx

```diff
@@ -129,27 +161,76 @@ const PaymentBuilderWidget: FC<PaymentBuilderWidgetProps> = ({ action }) => {
     },
     {
       key: ExpenditureStep.Funding,
-      heading: { label: formatText({ id: 'expenditure.fundingStage.label' }) },
+      heading: {
+        label: formatText({ id: 'expenditure.fundingStage.label' }),
+        decor:
+          expenditureStatus === ExpenditureStep.Funding &&
+          isFundingMotion &&
+          !fundingMotions?.[0]?.motionStateHistory.hasPassed &&
+          !isFundingMotionFailed &&
+          motionStakes ? (
+            <MotionCountDownTimer
+              motionState={motionState}
+              motionId={motionId}
+              motionStakes={motionStakes}
+              refetchMotionState={refetchMotionState}
+            />
+          ) : null,
+      },
       content:
         expenditureStatus === ExpenditureStep.Funding ? (
-          <StepDetailsBlock
-            text={formatText({
-              id: 'expenditure.fundingStage.info',
-            })}
-            content={
-              <Button
-                className="w-full"
-                onClick={showFundingModal}
+          <>
+            {isFundingMotion && !isFundingMotionFailed && !isFundingClaimed ? (
+              <div className="flex flex-col gap-2">
+                <RequestBox
+                  title={formatText({ id: 'expenditure.fundingRequest.title' })}
+                  actions={fundingMotions as ColonyMotion[]}
```

Comment:
Could this prop be renamed to `motions` to avoid confusion with `ColonyAction` type?